TF_Settings_Web/src/Components/Pages/Camera/QuickSetup/Calibration/CalibrationComponents.tsx

```diff
@@ -113,14 +129,15 @@ const ReturnToPositionScreenMessage: React.FC<HandsLostProps> = ({ timeToPosSele
     );
 };
 
-export const CalibrationHandLostMessage: React.FC<HandsLostProps> = ({ timeToPosSelect }) => {
+export const CalibrationHandLostMessage: React.FC<HandsLostProps> = ({ display, timeToPosSelect, handsLostStyle }) => {
```

Comment:
Should the CalibrationHandLostMessage in CalibrationCompleteScreen be updated to use these new props?

Reply:
It now does :+1: